src/hotspot/share/opto/superword.cpp

```diff
@@ -3526,12 +3526,11 @@ void SuperWord::packset_sort(int n) {
         Node_List* t = q_i;
         *(_packset.adr_at(i)) = q_low;
         *(_packset.adr_at(i-1)) = q_i;
-        swapped = true;
+        max_swap_index = i;
```

Comment:
So we not using `i+1` here because all previous values should be < than `i`'s
Right?

Reply:
Yes.
The last `i`'s value is > previous values and values between `i` and end are already sorted.